cmd/server/app/migrate_up.go

```diff
@@ -127,10 +130,64 @@ var upCmd = &cobra.Command{
 			return fmt.Errorf("error preparing authz client: %w", err)
 		}
 
-		return nil
+		store := db.NewStore(dbConn)
+		return migrateProjectsRootAndMetadata(ctx, cmd, store)
 	},
 }
 
+func migrateProjectsRootAndMetadata(ctx context.Context, cmd *cobra.Command, store db.Store) error {
+	cmd.Println("Migrating projects to have no parent...")
+	projs, err := store.ListNonOrgProjects(ctx)
+	if err != nil {
+		return fmt.Errorf("error while listing non-org projects: %w", err)
+	}
+
+	for _, p := range projs {
+		m := projects.NewSelfEnrolledMetadata()
+		if err := json.Unmarshal(p.Metadata, &m); err != nil {
+			return fmt.Errorf("error while marshalling old metadata: %w", err)
+		}
+
+		newmeta, err := json.Marshal(&m)
+		if err != nil {
+			return fmt.Errorf("error while marshalling new metadata: %w", err)
+		}
+
+		if p.ParentID.Valid {
```

Comment:
I think this logic is accidentally correct for our current dataset, since we haven't created any nested projects.  However, what it actually does is to remove all project nesting, not just the top-level org projects.

Also, it feels like we should be copying any entitlements from the root-level project to the subprojects, no?

Reply:
(That would involve iterating over the org-level projects, and then for each finding their children and releasing them.  Alternatively, you could do a self-join between root-level projects and their immediate descendants.)

Reply:
We never did anything on the org projects, there should be no entitlements for fga relationships.

Reply:
I think the entitlements were all done manually, so I'm not sure how they went.

If we didn't do openfga at the org level, I think it's worth a comment here explaining why it's not a concern, so if we copy the code for some other purpose in the future, we think about that sync.